packages/headless/src/controllers/product-listing/facet/headless-product-listing-facet.ts

```diff
@@ -106,6 +113,8 @@ export function buildFacet(
       dispatch(getAnalyticsActionForToggleFacetSelect(getFacetId(), selection));
     },
 
+    toggleExclude() {},
```

Comment:
Is that because the method is not yet implemented?

Reply:
Whoops, no. I think this was just when I was going around touching all of the files that needed to be updated and this one got left by the wayside 🤦 I'll quickly implement this now